### DISCUSSION TITLE
Suggestion: let TAs customize their default Teams message

### DISCUSSION THREAD

**Discussion by paigecache** [sandboxnu/office-hours#788] [Ideas]
This was a great idea for a feature to add! At this point in the semester though most of the students in OH are repeats, and the initial message I would want to send is different from the one that's auto-generated

**Comment by dfarooq610** [comment]
Hi! We do currently support the ability to customize your default teams message! It's in the Settings Page!

**Reply by paigecache** [reply]
Oh sick, did not notice that; glad that you have it!
